frontend/js/src/user/Listens.tsx

```diff
@@ -744,6 +752,17 @@ export default class Listens extends React.Component<
         />
       );
     }
+    if (canManuallyMap) {
+      additionalMenuItems.push(
+        <ListenControl
+          text="Map to a MusicBrainz Recording"
```

Comment:
This text is too long and goes over two lines. I removed "Manually" from it, but it could be made shorter

Reply:
changed to "Link to MusicBrainz" with a chain icon